runtime/README.md

```diff
@@ -281,6 +281,35 @@ Command Code: `0x5349_4756` ("SIGV")
 | chksum      | u32      | Checksum over other output arguments, computed by Caliptra. Little endian.
 | fips\_status | u32      | Indicates if the command is FIPS approved or an error.
 
+### LMS\_SIGNATURE\_VERIFY
+
+Verifies an LMS signature. The hash to be verified is taken from
+Caliptra's SHA384 accelerator peripheral.
+The supported parameter set is limited to: sha256/192, w=4, and h=15 (the same used for the Caliptra image)
+
+Command Code: `0x4C4D_5356` ("LMSV")
+
+*Table: `LMS_SIGNATURE_VERIFY` input arguments*
+
+| **Name**              | **Type** | **Description**
+| --------              | -------- | ---------------
+| chksum                | u32      | Checksum over other input arguments, computed by the caller. Little endian.
+| pub\_key\_tree\_type  | u8[4]    | LMS public key algorithm type. Must equal 12.
+| pub\_key\_ots\_type   | u8[4]    | LM-OTS algorithm type. Must equal 7.
+| pub\_key\_id          | u8[16]   | Private key identifier
+| pub\_key\_digest      | u8[24]   | Public key hash value
+| signature\_q          | u8[4]    | Leaf of the Merkle tree where the OTS public key appears
+| signature\_ots        | u8[1252] | LM-OTS signature
+| signature\_tree\_type | u8[4]    | LMS signature Algorithm type. Must equal 12.
+| signature\_tree\_path | u8[360]  | Path through the tree from the leaf associated with the LM-OTS signature to the root
```

Comment:
I wonder if we should define the structure of the signature in a separate table and then just have a flat `signature` field in the command.

I don't feel too strongly though.

Reply:
This crossed my mind at one point to. The only reason I didn't do it is because I didn't think we had done multiple layers of structs anywhere else. I am okay with it either way.

Reply:
Ok to leave it since this aligns with how the ECDSA command works.